scripts/check-siblings.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const path = require('path');
+
+//Filters all Sentry packages but Capacitor itself.
+const jsonFilter = /\s*\"\@sentry\/(?!capacitor)(?<packageName>[a-zA-Z]+)\"\:\s*\"(?<version>.+)\"/;
+const siblingVersion = "7.15.0";
```

Comment:
Can we just grab this from the sentry capacitor package.json?

Reply:
Yes we can

Reply:
Another option is to grab it from the environment inside of the postinstall script:
```
console.log("Core version is : "+ env.npm_package_dependencies__sentry_browser);
output
Core version is : 7.15.0
```
Not sure about the compatibility about it with npm or older yarn versions.